core/src/test/scala/besom/internal/DecoderTest.scala

```diff
@@ -138,20 +139,20 @@ class DecoderTest extends munit.FunSuite:
         then
           Map(
             "foo" -> 10.asValue.asOutputValue(isSecret = false, Nil),
-            "bar" -> List("qwerty".asValue).asValue.asOutputValue(isSecret = false, Nil),
+            "bar" -> Iterable("qwerty".asValue).asValue.asOutputValue(isSecret = false, Nil),
```

Comment:
are these changes in tests necessary? Lists are <: Iterable

Reply:
Nope. These changes were necessary before I created encoders and decoders for list. (iterable decoder and encoder was not enough).